modules/ephemeris/src/test/scala/gem/horizons/VelocityCompressionSpec.scala

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.horizons
+
+import gem.EphemerisKey
+import gem.enum.Site.GN
+import gem.math.Ephemeris
+import gem.test.RespectIncludeTags
+import gem.test.Tags._
+
+import VelocityCompression._
+
+import cats.effect.IO
+import cats.implicits._
+import cats.tests.CatsSuite
+
+import java.time.{ LocalDate, Month }
+import java.time.temporal.ChronoUnit.MINUTES
+
+@SuppressWarnings(Array("org.wartremover.warts.Equals"))
+final class VelocityCompressionSpec extends CatsSuite with EphemerisTestSupport with RespectIncludeTags {
+  import VelocityCompressionSpec._
+
+  test("compresses") {
+    val a = stream("borrelly")
+              .through(EphemerisParser.elements[IO])
+              .through(standardVelocityCompression)
+
+    val e = stream("borrelly-compressed")
+              .through(EphemerisParser.elements[IO])
+
+    val actual   = a.runLog.unsafeRunSync
+    val expected = e.runLog.unsafeRunSync
+
+    assert(actual == expected)
+  }
+
+  // Andy's result: 99%
+  test("2014 UR compresses to 79% of original size", RequiresNetwork) {
+    val start = LocalDate.of(2015, Month.SEPTEMBER, 15)
+    val end   = LocalDate.of(2015, Month.NOVEMBER,  15)
+
+    testCompression("2014 UR", ur_2014, 0.79, start, end)
```

Comment:
This is the test case that differs the most from Andy's results.  I need to see the exact test setup he used to figure out what is going on.